Backend/Settings/PathSettings.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FRITeam.Swapify.Backend.Settings
+{
+    public class PathSettings : SettingsBase
+    {
+
+        public string CoursesJsonPath { get; set; }
+
+        public PathSettings()
+        {
+
+        }
+
+        public override void Validate()
+        {
+            if (string.IsNullOrEmpty(CoursesJsonPath))
+                Errors.AppendLine($"Setting {nameof(CoursesJsonPath)} is missing in {nameof(PathSettings)} configuration section.");
```

Comment:
did you test it before push? This will not work because you forgot to call CheckErrors() method and you also forgot to register PathSettings in startup file... so this code will never execute.